public/app/features/dashboard/components/DashNav/common_tools.ts

```diff
@@ -3,8 +3,6 @@
 import _ from 'lodash';
 import moment from 'moment'; // eslint-disable-line no-restricted-imports
 
-const GLOBAL = 'global';
```

Comment:
This was already removed in https://github.com/digiaiiris/grafana/pull/140/files. Just a side note ;)